<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[<div "featureFile">Feature File: HeatclinicLogin.feature</div>]]>
    </line>
    <line>
      <![CDATA[<div "feature">Feature: Broadleaf Commerce Login Functionality</div>]]>
    </line>
    <line>
      <![CDATA[<div "scenario">Scenario: 1. Invalid user valid password</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Given  As a not validated user (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">When  Browse to the &quot;http:&#x2F;&#x2F;demo.broadleafcommerce.org&#x2F;&quot; (passed : 1s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Then  Broadlief home page should show (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">When  User click login button (passed : 2s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">And  Enter &quot;iivaan_NotValid@Gmail.com&quot; as emaill address (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">And  Enter &quot;ameera&quot; as password (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">And  Click login (passed : 0s)</div>]]>
    </line>
    <line>
      <![CDATA[<div "result">Then  Invalid email or password message display as &quot;The ex-mail address and&#x2F;or password entered do not match our records. Please try again&quot; (failed : 0s)</div>]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="8974" started-at="2015-01-17T19:55:17Z" finished-at="2015-01-17T19:55:26Z">
    <groups>
      <group name="CucumberTest">
        <method signature="HeatClinicTestRunner.cucumberTest()[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="cucumberTest" class="com.paxovision.cucumber.HeatClinicTestRunner"/>
      </group> <!-- CucumberTest -->
    </groups>
    <test name="Default test" duration-ms="8974" started-at="2015-01-17T19:55:17Z" finished-at="2015-01-17T19:55:26Z">
      <class name="com.paxovision.cucumber.HeatClinicTestRunner">
        <test-method status="PASS" signature="springTestContextPrepareTestInstance()[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="springTestContextPrepareTestInstance" is-config="true" duration-ms="3356" started-at="2015-01-17T14:55:17Z" depends-on-methods="org.springframework.test.context.testng.AbstractTestNGSpringContextTests.springTestContextBeforeTestClass" finished-at="2015-01-17T14:55:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextPrepareTestInstance -->
        <test-method status="PASS" signature="springTestContextBeforeTestClass()[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="springTestContextBeforeTestClass" is-config="true" duration-ms="0" started-at="2015-01-17T14:55:17Z" finished-at="2015-01-17T14:55:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestClass -->
        <test-method status="FAIL" signature="cucumberTest()[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="cucumberTest" duration-ms="5602" started-at="2015-01-17T14:55:20Z" description="TestNGCucumberRunner to invoke Cucumber" finished-at="2015-01-17T14:55:26Z">
          <exception class="cucumber.runtime.CucumberException">
            <message>
              <![CDATA[java.lang.AssertionError: 
Expected: equalToIgnoringCase("The ex-mail address and/or password entered do not match our records. Please try again")
     but: was The e-mail address and/or password entered do not match our records. Please try again]]>
            </message>
            <full-stacktrace>
              <![CDATA[cucumber.runtime.CucumberException: java.lang.AssertionError: 
Expected: equalToIgnoringCase("The ex-mail address and/or password entered do not match our records. Please try again")
     but: was The e-mail address and/or password entered do not match our records. Please try again
	at cucumber.api.testng.TestNGCucumberRunner.runCukes(TestNGCucumberRunner.java:48)
	at com.paxovision.cucumber.HeatClinicTestRunner.cucumberTest(HeatClinicTestRunner.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.MethodInvocationHelper$1.runTestMethod(MethodInvocationHelper.java:200)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:172)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeHookable(MethodInvocationHelper.java:212)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1224)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1149)
	at org.testng.TestNG.run(TestNG.java:1057)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: java.lang.AssertionError: 
Expected: equalToIgnoringCase("The ex-mail address and/or password entered do not match our records. Please try again")
     but: was The e-mail address and/or password entered do not match our records. Please try again
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at com.paxovision.cucumber.steps.HeatclinicSteps.invalid_email_or_password_message_display_as(HeatclinicSteps.java:62)
	at ?.Then Invalid email or password message display as "The ex-mail address and/or password entered do not match our records. Please try again"(HeatclinicLogin.feature:12)
]]>
            </full-stacktrace>
          </exception> <!-- cucumber.runtime.CucumberException -->
          <reporter-output>
            <line>
              <![CDATA[<div "featureFile">Feature File: HeatclinicLogin.feature</div>]]>
            </line>
            <line>
              <![CDATA[<div "feature">Feature: Broadleaf Commerce Login Functionality</div>]]>
            </line>
            <line>
              <![CDATA[<div "scenario">Scenario: 1. Invalid user valid password</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Given  As a not validated user (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">When  Browse to the &quot;http:&#x2F;&#x2F;demo.broadleafcommerce.org&#x2F;&quot; (passed : 1s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Then  Broadlief home page should show (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">When  User click login button (passed : 2s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">And  Enter &quot;iivaan_NotValid@Gmail.com&quot; as emaill address (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">And  Enter &quot;ameera&quot; as password (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">And  Click login (passed : 0s)</div>]]>
            </line>
            <line>
              <![CDATA[<div "result">Then  Invalid email or password message display as &quot;The ex-mail address and&#x2F;or password entered do not match our records. Please try again&quot; (failed : 0s)</div>]]>
            </line>
          </reporter-output>
        </test-method> <!-- cucumberTest -->
        <test-method status="PASS" signature="springTestContextBeforeTestMethod(java.lang.reflect.Method)[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="springTestContextBeforeTestMethod" is-config="true" duration-ms="0" started-at="2015-01-17T14:55:20Z" finished-at="2015-01-17T14:55:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.paxovision.cucumber.HeatClinicTestRunner.cucumberTest() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextBeforeTestMethod -->
        <test-method status="PASS" signature="springTestContextAfterTestMethod(java.lang.reflect.Method)[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="springTestContextAfterTestMethod" is-config="true" duration-ms="16" started-at="2015-01-17T14:55:26Z" finished-at="2015-01-17T14:55:26Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.paxovision.cucumber.HeatClinicTestRunner.cucumberTest() throws java.io.IOException]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestMethod -->
        <test-method status="PASS" signature="springTestContextAfterTestClass()[pri:0, instance:com.paxovision.cucumber.HeatClinicTestRunner@18c14dd1]" name="springTestContextAfterTestClass" is-config="true" duration-ms="0" started-at="2015-01-17T14:55:26Z" finished-at="2015-01-17T14:55:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- springTestContextAfterTestClass -->
      </class> <!-- com.paxovision.cucumber.HeatClinicTestRunner -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
